build.env

```diff
@@ -20,7 +20,7 @@ GOLANG_VERSION=1.16.4
 GO111MODULE=on
 
 # commitlint version
-COMMITLINT_VERSION=12.1.4
+COMMITLINT_VERSION=13.1.0
```

Comment:
Is it possible to use a tag like `latest`? We would like to use the most current version by default,  until it breaks :-)

Reply:
Yes updated